File: OP_PROJECT_JAVA/src/game/Plants/Belladonna.java
package game.Plants;

import game.Plant;
import game.Position;
import game.World;

public class Belladonna extends Plant {
    private static final int BELLADONNA_STRENGTH = 99;
    private static final String BELLADONNA_TYPE = "Belladonna";


    public Belladonna(Position position, World world) {
        super(BELLADONNA_TYPE, BELLADONNA_STRENGTH, position, world);
    }
}

File: OP_PROJECT_PYTHON/src/game/Plants/Belladonna.py
from src.game.Plant import Plant


class Belladonna(Plant):
    BELLADONNA_STRENGTH = 99
    BELLADONNA_TYPE = "Belladonna"

    def __init__(self, position, world):
        super().__init__(Belladonna.BELLADONNA_TYPE, Belladonna.BELLADONNA_STRENGTH, position, world)
